bin/node-template/node/Cargo.toml

```diff
@@ -31,9 +32,16 @@ sp-consensus = { version = "0.8.0-rc5", path = "../../../primitives/consensus/co
 sc-consensus = { version = "0.8.0-rc5", path = "../../../client/consensus/common" }
 sc-finality-grandpa = { version = "0.8.0-rc5", path = "../../../client/finality-grandpa" }
 sp-finality-grandpa = { version = "2.0.0-rc5", path = "../../../primitives/finality-grandpa" }
+sc-rpc = { version = "2.0.0-rc5", path = "../../../client/rpc" }
+sp-api = { version = "2.0.0-rc5", path = "../../../primitives/api" }
+sc-rpc-api = { version = "0.8.0-rc5", path = "../../../client/rpc-api" }
 sc-client-api = { version = "2.0.0-rc5", path = "../../../client/api" }
+
 sp-runtime = { version = "2.0.0-rc5", path = "../../../primitives/runtime" }
-sc-basic-authorship = { path = "../../../client/basic-authorship", version = "0.8.0-rc5"}
+sp-blockchain = { version = "2.0.0-rc5", path = "../../../primitives/blockchain" }
+sp-block-builder = { version = "2.0.0-rc5", path = "../../../primitives/block-builder" }
+sc-basic-authorship = { version = "0.8.0-rc5", path = "../../../client/basic-authorship" }
+substrate-frame-rpc-system = { version = "2.0.0-rc5", path = "../../../utils/frame/rpc/system" }
```

Comment:
I recommend you group all the new dependencies into a seperate block below the existing dependencies, and label that block with a comment `# imports for rpc` or something like that